library/src/main/java/com/rtchagas/pingplacepicker/repository/googlemaps/GoogleMapsRepository.kt

```diff
@@ -122,10 +131,127 @@ class GoogleMapsRepository constructor(
                     if (("OK" == result.status) && result.results.isNotEmpty()) {
                         return@flatMap getPlaceById(result.results[0].placeId)
                     }
-                    return@flatMap Single.just(null)
+                    return@flatMap Single.just(PlaceFromCoordinates(location.latitude, location.longitude))
```

Comment:
Yes, this was my mistake. We can't push null values in RxJava.
Now that the function always return a Place, please change the return type to Single<**Place**>